server/lib/nl/explore/utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper variables such as prompts for Explore"""
+
+_RETRIES = 3
+
+_GEMINI_MODEL = "gemini-2.5-pro-preview-06-05"
+
+_RELATED_QUESTIONS_QUERY = """
```

Comment:
nit: RELATED_QUESTIONS_PROMPT

nit: since these are only used by one file, I don't think we need this utils file and these should jsut live in the file they're used in

nit: naming constants with a "_" prefix by convention indicates a private constant that should only be used within that file